Lib/fontTools/ttLib/tables/TupleVariation.py

```diff
@@ -48,7 +48,7 @@ def __eq__(self, other):
 		return self.coordinates == other.coordinates and self.axes == other.axes
 
 	def getUsedPoints(self):
-		return set([i for i,p in enumerate(self.coordinates) if p is not None])
+		return frozenset([i for i,p in enumerate(self.coordinates) if p is not None])
```

Comment:
same for `frozenset(i for i, p ...)` it also takes a generator expression

Reply:
same. is slower:
```
In [2]: timeit set([_ for _ in l])
2.12 µs ± 6.44 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [3]: timeit set(_ for _ in l)
2.76 µs ± 10.8 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [4]: timeit set(l)
821 ns ± 2.35 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

Frozenset is very marginally slower than set; but that's because it does hashing.